### PR TITLE
feat: Allow configuration with 'values[...]' and 'set{...}'

### PR DESCRIPTION
**Pull request by kpshjk**
@dploeger 
As requested a PR... I'll call you monday

### CODE REVIEW

File: vars.tf [hunk 1]
```diff
@@ -21,3 +21,19 @@ variable "cluster-issuers-yaml" {
...
 }
+
+variable "set-list" {
+  type = list(object({
+    name = string,
+    value = string,
+    type = string,
+  }))
+  default = []
+  description = "A list of settings to apply to on the helm chart using its parameter 'set { }' "
```
Comment (dploeger) [inline vars.tf]:
```suggestion
  description = "A list of additional settings to apply to the helm chart using the terraform `set{}` parameter"
```

File: vars.tf [hunk 2]
```diff
@@ -21,3 +21,19 @@ variable "cluster-issuers-yaml" {
...
+}
+
+variable "values" {
+  type = list(string)
+  default = []
+  description = "A list of values to apply on the helm chart using its parameter 'values = [...]' "
```
Comment (dploeger) [inline vars.tf]:
```suggestion
  description = "A list of additional values to apply to the helm chart using the  'values = [...]' parameter"
```

**Review by dploeger — CHANGES_REQUESTED** [review]
I have two suggestions about the documentation. The rest looks fine.

**Comment by kpshjk** [comment]
@dploeger 
I finally succeeded refactoring the PR accordingly.

**Review by dploeger — APPROVED** [review]
Wonderful. Thanks! 🎁

**Pull request merged**